pkgs/development/libraries/lmdb/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname = "lmdb";
-  version = "0.9.28";
+  version = "0.9.29";
 
   src = fetchgit {
```

Comment:
This could be converted to fetchFromGitLab.

Reply:
Sure, but I hope that doesn't need to hold this up

Reply:
Also please update the homepage to https://symas.com/lmdb/technical/